app/controllers/spree/retail/shopify/hooks/refunds_controller.rb

```diff
@@ -0,0 +1,24 @@
+module Spree
+  module Retail
+    module Shopify
+      module Hooks
+        class RefundsController < HooksController
+          def create
```

Comment:
I obviously initially thought that it was not necessary thus why I didn't implement it, but I see your point and I'll change it to make it idempotent.